tests/transactional/disable_selinux.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: transactional-update
+# Summary: Disable SELinux on transactional server
+#
+# Maintainer: QA-C team <qa-c@suse.de>
+
+
+
+
+
+
+use base "consoletest";
+use testapi;
+use strict;
+use warnings;
+use transactional qw(process_reboot);
+use version_utils qw(is_leap_micro is_microos is_sle_micro is_public_cloud);
+use Utils::Systemd qw(systemctl);
+use Utils::Logging 'save_and_upload_log';
+use transactional qw(trup_call check_reboot_changes process_reboot);
+use publiccloud::utils qw(allow_openqa_port_selinux);
```

Comment:
Just a small hint (not really important): When you only import one identifier of that namespace, you can choose:
`use <NAMESPACE> '<IDENTIFIER>'` instead of using quote words (qw()). It is a little shorter and might be easier to read. In your code, this is a bit inconsistent.